Simplenote/Snackbar.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  Snackbar.swift
+//  Simplenote
+//
+//  Created by Kevin LaCoste on 2020-11-09.
+//  Copyright © 2020 Automattic. All rights reserved.
+//
+
+import Foundation
+
+class Snackbar {
+	
+	var message: String
+	
+	static var presenter = SnackbarPresenter()
```

Comment:
Please review this singleton (a better spot would be, perhaps, the SnackbarPresenter class itself)

Reply:
Not sure what I was thinking here. I've moved it to the appropriate place now!